composer.json

```diff
@@ -41,7 +42,11 @@
     "config": {
         "optimize-autoloader": true,
         "preferred-install": "dist",
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true,
+            "pixelfear/composer-dist-plugin": true
+        }
```

Comment:
Please remove, it should go in your project instead.

Reply:
You get the issue when you run `composer update` in the project file, which I did when I needed to run the tests (and therefore needed all the packages)

Reply:
Ah you're absolutely right, I guess I haven't done a composer update in cms locally in a while. This PR isn't the right place for it to be added though. I'll add it separately. Don't do anything, it'll get resolved on merge. 👍

Reply:
If you merge 3.2 back into this branch, this chunk should disappear from the diff.